### ISSUE TITLE
decompress property is missing on AxiosRequestConfig interface

### ISSUE THREAD

**Issue by razafinarivohanania** [axios/axios#2990]
**Describe the bug**
Hello, i try to use axios with TypeScript but decompress property is absent on AxiosRequestConfig interface.

```ts
export interface AxiosRequestConfig {
  url?: string;
  method?: Method;
  baseURL?: string;
  transformRequest?: AxiosTransformer | AxiosTransformer[];
  transformResponse?: AxiosTransformer | AxiosTransformer[];
  headers?: any;
  params?: any;
  paramsSerializer?: (params: any) => string;
  data?: any;
  timeout?: number;
  timeoutErrorMessage?: string;
  withCredentials?: boolean;
  adapter?: AxiosAdapter;
  auth?: AxiosBasicCredentials;
  responseType?: ResponseType;
  xsrfCookieName?: string;
  xsrfHeaderName?: string;
  onUploadProgress?: (progressEvent: any) => void;
  onDownloadProgress?: (progressEvent: any) => void;
  maxContentLength?: number;
  validateStatus?: (status: number) => boolean;
  maxRedirects?: number;
  socketPath?: string | null;
  httpAgent?: any;
  httpsAgent?: any;
  proxy?: AxiosProxyConfig | false;
  cancelToken?: CancelToken;
}
```
But as mentionned on README, we can modify decompress value with true or false.

```js
{
  //other options

  // `decompress` indicates whether or not the response body should be decompressed 
  // automatically. If set to `true` will also remove the 'content-encoding' header 
  // from the responses objects of all decompressed responses
  // - Node only (XHR cannot turn off decompression)
  decompress: true // default
}
```
**To Reproduce**
Just make a simple request with decompress option.

```ts
import axios from 'axios';

(async () => {
    const response = await axios.request({
        url: 'https://www.google.com',
        decompress: false
    });
})();
```

Then compile code with tsc, and this error occured :

```sh
src/connection.ts:6:9 - error TS2345: Argument of type '{ url: string; decompress: boolean; }' is not assignable to parameter of type 'AxiosRequestConfig'.
  Object literal may only specify known properties, and 'decompress' does not exist in type 'AxiosRequestConfig'.

6         decompress: false
          ~~~~~~~~~~~~~~~~~
```

**Expected behavior**
Compilation success

**Environment:**
 - Axios : 0.19.2
 - OS: Ubuntu 18.04.4
 - Browser : Chrome
 - Browser Version : 80.0.3987.149

**Additional context/Screenshots**

![a](https://user-images.githubusercontent.com/24853543/83329899-a11b4580-a294-11ea-86d7-218ae87788e5.jpg)

**Comment by chinesedfan** [comment]
It was [there](https://github.com/axios/axios/blob/master/index.d.ts#L73). Note the master has more commits than 0.19.2, which will be released in 0.20. See https://github.com/axios/axios/projects/4 to track the release progress.

**Comment by razafinarivohanania** [comment]
@chinesedfan Thanks, i'll wait for next release. :)
